staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/passwordfile.go

```diff
@@ -60,14 +62,31 @@ func NewCSV(path string) (*PasswordAuthenticator, error) {
 		if err != nil {
 			return nil, err
 		}
-		if len(record) < 3 {
+		size := len(record)
+		if size < 3 ||
+			(size == 3 && len(record[2]) == 0) {
 			return nil, fmt.Errorf("password file '%s' must have at least 3 columns (password, user name, user uid), found %d", path, len(record))
 		}
+
+		if size > 3 &&
+			record[size-1] == "" &&
+			record[size-2] == "" {
+			return nil, fmt.Errorf("token file '%s' is expected to specify tenant but it doesn't, found %d", path, len(record))
+		}
+
+		tenant := metav1.TenantSystem
+		// check for comma in the last record
+		if size >= 3 &&
+			record[size-1] != "" &&
+			record[size-2] == "" {
```

Comment:
I see both styles of record[i] == "" and len(record[i]) == 0 in this file. Can you stick to one style?